spec/factories/restaurant_factory.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                    :uuid             not null, primary key
+#  accept10bis           :boolean          default(FALSE), not null
+#  address               :string
+#  cuisine               :string
+#  latitude              :float
+#  longitude             :float
+#  max_delivery_time_min :integer
+#  name                  :string           not null
+#  rating                :float            default(0.0)
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#
+require 'faker'
+FactoryBot.define do
+  factory :restaurant do
+    name { Faker::Restaurant.name }
+    cuisine { Restaurant::E_CUISINES.sample }
+  end
+  factory :restaurant_no_cuisine do
```

Comment:
see [Traits](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits) for this kind of a behaviour